src/main/java/core/basesyntax/service/operationwithfruits/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.operationwithfruits;
+
+public interface OperationHandler {
+    default Integer getOperation(Integer count) {
```

Comment:
```suggestion
    void handle(FruitTransacation transaction) {
```
![image](https://user-images.githubusercontent.com/12864128/192001671-377c1f81-5043-4f50-8829-d3cf7e621dcb.png)